docs/service_x509_credentials_aws.md

```diff
@@ -0,0 +1,346 @@
+      "account": "<aws account id associated with domain name>"
+    }
+  ]
+}
+```
+
+The domain/account/service values here must match to the IAM Role
+created earlier. For example, if the service identity name is `api`
+in domain `sports` whose corresponding aws account id is `123456789`,
+then the `sia_config` file will be as follows:
+
+```
+{
+  "version": "1.0.0",
+  "service": "api",
+  "accounts": [
+    {
+      "domain": "sports",
+      "account": "123456789"
+    }
+  ]
+}
+```
+
+If you require bastion ssh access to your instances, then the profile
+role must be called `<domain>.<service>-service` and you don't need to
+specify this configuration file unless you want to change the ownership
+of the generated private key and certificate files - see
+[Private Key File Ownership](#private-key-file-ownership) section for more details.
+
+### Private Key File Ownership
+
+By default SIA agent runs as root and the private key is only readable
+by root. If your service is running as another user, you can configure
+SIA agent to automatically change the ownership of the private key file
+to the configured user.
+For example, if you want the private key to be owned by `ec2-user`, then
+your `sia_config` configuration file would be as follows:
+
+```
+{
+  "version": "1.0.0",
+  "service": "api",
+  "accounts": [
+    {
+      "domain": "sports",
+      "account": "123456789",
+      "user": "ec2-user"
+    }
+  ]
+}
+```
+
+### SSH Certificate Support
+
+As part of the bootstrap process, SIA also retrieves a host ssh certificate
+to be used by the bastion daemon that provides ssh access to instances.
+However, if the administrator wants to disable this functionality, they
+can do so by specifying `ssh` entry in their `sia_config` file with `false`
+value as follows:
+
+```
+{
+  "version": "1.0.0",
+  "service": "api",
+  "ssh": false,
+  "accounts": [
+    {
+      "domain": "sports",
+      "account": "123456789"
+    }
+  ]
+}
+```
+
+## Private Key and Certificate Setup
+
+By default, the private key for the service identity is available in
+the `/var/lib/sia/keys` directory and has the name `<domain>.<service>.key.pem`.
+The private key is in PKCS#1 format. The corresponding X.509 certificate
+is in the `/var/lib/sia/certs` directory and has the name
+`<domain>.<service>.cert.pem`. The certificate is valid for 30 days and will
+be refreshed automatically by SIA every day. It is the responsibility of the
+application owner to update their container/application to refresh and use
+the latest certificate before it expires. In the same `/var/lib/sia/certs`
+directory SIA will also generate the Athenz CA certificate file called `ca.cert.pem`.
+
+### Role Certificate Support
+
+In addition to requesting Athenz service identity certificate, SIA provides
+the capability to request Athenz Role Certificates as well. If you want to change the default behavior or request
+SIA to automatically retrieve role certificates for your service, as part
```

Comment:
Removed all references to ZTokens used within Yahoo data centers and internal Sia packages. Original text:
 the capability to request Athenz Role Certificates as well. These replace
 the Access Tokens (ZTokens) used within Yahoo Data Centers though they’re
 issued for a single role only. If you want to change the default behavior or request
333a344,360
 ## Installing SIA without Foundation Image

 If your property is not able to use the foundation image that comes pre-installed
 with SIA, you can download and include the SIA rpm (available for RHEL and
 CentOS) as part of your image. The rpms are available form our Artifactory in AWS:

 RHEL/Centos:
 - v7 [https://artifactory.ouroath.com/artifactory/athens_rpms/rhel/7Server/current/x86_64/Packages/](https://artifactory.ouroath.com/artifactory/athens_rpms/rhel/7Server/current/x86_64/Packages/)

 Ubuntu:
 - Bionic [https://artifactory.ouroath.com/artifactory/athenz_debs/pool/bionic/](https://artifactory.ouroath.com/artifactory/athenz_debs/pool/bionic/)
 - Xenial [https://artifactory.ouroath.com/artifactory/athenz_debs/pool/xenial/](https://artifactory.ouroath.com/artifactory/athenz_debs/pool/xenial/)
 - Trusty [https://artifactory.ouroath.com/artifactory/athenz_debs/pool/trusty/](https://artifactory.ouroath.com/artifactory/athenz_debs/pool/trusty/)

 The rpm package will install the binary and setup the sia service. The property
 must still follow all the steps  documented in this document.